test/comparators.go

```diff
@@ -0,0 +1,62 @@
+package test
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+// ErrorTypeComparator defines an error type comparator for use with go-cmp
```

Comment:
I'd put this in a `ExampleErrorTypeComparator` clause in the test suite so that it's vetted on build.

Reply:
there __is__ a `ExampleErrorTypeComparator` clause in the test file.